Makefile.sdk.mk

```diff
@@ -287,6 +291,12 @@ endif
 ifeq ($(host_arch), arm)
 	v8_arch := arm
 	android_target_platform := 14
+	v8_arm_abifloat := -D armfloatabi=softfp
```

Comment:
Let's name this one `v8_abi_flags` for consistency, and also to allow us to use this for other ABI-related flags for other architectures.